### ISSUE TITLE
[New Model]: Emu3

### ISSUE THREAD

**Issue by kuangdao** [vllm-project/vllm#11008]
### The model to consider.

https://huggingface.co/BAAI/Emu3-Chat

### What's your difficulty of supporting the model you want?

i want to know when will support Emu3 model.

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by hmellor** [comment]
The LM part of Emu3 should be supported by the Transformers backend. This could be a good candidate for adding VLM support to the Transformers backend.

cc @zucchini-nlp

**Comment by zucchini-nlp** [comment]
Yeah, Emu3 doesn't have much special processing and should be straightforward as LLaVA. I can add it in the first batch of models, but I am still waiting on review for transformers-side changes. We need to add a few helpers and standardize model code, before supporting vLLM

**Comment by hongjx175** [comment]
Looking forward to update
